priv/repo/migrations/20181011082425_update_timestamp_fields.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Hexpm.RepoBase.Migrations.UpdateTimestampFields do
+  use Ecto.Migration
+
+  defp fixup(table, columns) do
+    alter table(table) do
+      for column <- columns do
+        modify(column, :utc_datetime_usec)
```

Comment:
What postgres types are we changing between here?

Reply:
We're changing from `TIMESTAMP(0)` to `TIMESTAMP`